ssh/tailssh/tailssh.go

```diff
@@ -1519,8 +1519,9 @@ func (ss *sshSession) startNewRecording() (_ *recording, err error) {
 
 	now := time.Now()
 	rec := &recording{
-		ss:    ss,
-		start: now,
+		ss:       ss,
+		start:    now,
+		failOpen: onFailure == nil || onFailure.TerminateSessionWithMessage == "",
```

Comment:
Why this second conditional? Can we make a nil callback be how fail open is communicated?

Reply:
Nil callback is useful for when the recording fails to start. This is for when it starts and then fails.